crates/polars-plan/src/dsl/mod.rs

```diff
@@ -1208,7 +1208,7 @@ impl Expr {
             self.apply_many_private(
                 FunctionExpr::RollingExpr(rolling_function_by(options)),
                 &[col(&name)],
-                true,
+                false,
```

Comment:
Ai, good catch. That took a long time.

Reply:
Rolling functions return something which is the same length as the input, so I don't think they should ever return a scalar

Reply:
ah thanks - will take a look, have made a note-to-self